main/seccomp.c

```diff
@@ -44,6 +44,8 @@ int installSyscallFilter (void)
 	// main/parse.c:2764 : tagFilePosition (&tagfpos);
 	seccomp_rule_add (ctx, SCMP_ACT_ALLOW, SCMP_SYS (lseek), 0);
 
+	seccomp_rule_add (ctx, SCMP_ACT_ALLOW, SCMP_SYS (getpid), 0);
```

Comment:
are you sure this is right? 

http://cs.bazel.build/search?q=r%3Actags+getpid+&num=50

I don't think ctags does a getpid call? If it does, document where it comes from.